website/docs/features/responsible_ai/Data Balance Analysis.md

```diff
@@ -19,7 +19,7 @@ Fairness-related harms include:
 * **Quality of service harms**: When an AI system does not work as well for one group of people as it does for another.
 * **Stereotyping harms**: When an AI system makes unfair generalizations about groups of people and reinforces negative stereotypes.
 * **Demeaning harms**: When an AI system is actively derogatory or offensive.
-* **Over- and underrepresentation harms**: When an AI system over- or underrepresents some groups of people or may even erase some groups entirely.
+* **Over- and underrepresentation harms**: When an AI system over- or under represents some groups of people or may even erase some groups entirely.
```

Comment:
This isn't correct since the over has a hyphen, and underrepresent is a word, while "under represent" isn't really a term we'd use. Acrolinx is just wrong on this one.

Reply:
Thanks!